sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/README.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
The heading should be at the very top. Be sure to install the `markdown-lint` extension.